### ISSUE TITLE
third party applications and Big Sur

### ISSUE THREAD

**Issue by SebastianZiegler** [SafeExamBrowser/seb-mac#78]
Hello,

 has anybody get this to work. 

Try this:

https://safeexambrowser.org/macosx/mac_usermanual_en.html#secure_environment

but it seems that in Big Sur Parental Control Changed

Any idea?

best regards
Sebastian

**Comment by danschlet** [comment]
Yes, unfortunately Apple removed Simple Finder and changed the whole parental controls. We have to investigate if it is still possible to use SEB for macOS in a similar way with third party applications. It might be necessary to configure the separate macOS user account using an MDM system like Jamf Pro to achieve something similar like before with Parental Controls and Simple Finder (or it might not be possible at all). We recommend to use a Windows virtual desktop environment for third party applications and just display it on Macs in a full screen SEB browser window (for example possible with VMware Horizon HTML View, but possibly also with cloud virtual desktops from Amazon and Microsoft).

**Comment by danschlet** [comment]
See this discussion: https://github.com/SafeExamBrowser/seb-mac/discussions/281
